### ISSUE TITLE
Dropdown active state mobile and desktop

### ISSUE THREAD

**Issue by meegido** [huridocs/uwazidocs#4833]
**Describe the bug**
Dropdown active state doesn't work so the user is not able to see that they are in one of the belonging pages. 

**To Reproduce**
1. Go to any instance, for example 
2. Click on a menu item with dropdown. 
3. Focus out the mouse from there. 
   3.4. On mobile, go again to the menu item you clicked. 
4. See the error: There's no visual indication about that you are in any of the pages inside the dropdown. 

**Expected behavior**
@juanmnl will provide the solution for this specific interaction design. 

**Screenshots**
[Video recording](https://watch.screencastify.com/v/FbTH5QYqlIm4DLVPf5hN)

**Device (please select all that apply)**
 - [ x] Desktop
 - [ x] Mobile

**Browser**
Firefox, Chrome, Safari, etc...

**Additional context**
It's important to check if the change is going to require fixes on every instance with public pages.

**Comment by aphilop** [comment]
@konzz I assume this is/would be fixed with the new top menu, correct?

**Comment by konzz** [comment]
Yes, but only where we use it, its not clear yet where or when we are going to release it
